### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Deeksha2501**
These are the some grammar checks I have done in README file
I hope after this no grammar issue will be left in this file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ Overall, Lightning guarantees rigorously tested, correct, modern best practices
 ## How flexible is it?
 As you see, you're just organizing your PyTorch code - there's no abstraction.
 
-And for the stuff that the Trainer abstracts out you can [override any part](https://pytorch-lightning.readthedocs.io/en/latest/introduction_guide.html#extensibility) you want to do things like implement your own distributed training, 16-bit precision, or even a custom backwards pass.
+And for the stuff that the Trainer abstracts out, you can [override any part](https://pytorch-lightning.readthedocs.io/en/latest/introduction_guide.html#extensibility) you want to do things like implementing your own distributed training, 16-bit precision, or even a custom backward pass.
```
Comment (williamFalcon) [inline README.md:123]: it really should be implement here.
1. endings of list items should match (running, swimming, etcing) (especially gerunds)

Comment (williamFalcon) [inline README.md:123]:
```suggestion
And for the stuff that the Trainer abstracts out, you can [override any part](https://pytorch-lightning.readthedocs.io/en/latest/introduction_guide.html#extensibility) you want to do things like implement your own distributed training, 16-bit precision, or even a custom backward pass.
```
```suggestion
And for the stuff that the Trainer abstracts out, you can [override any part](https://pytorch-lightning.readthedocs.io/en/latest/introduction_guide.html#extensibility) you want to do things like implement your own distributed training, 16-bit precision, or even a custom backward pass.
```

File: README.md [hunk 2]
```diff
@@ -226,7 +226,7 @@ In summary, you:
 [Check out the COLAB demo here](https://colab.research.google.com/drive/1F_RNcHzTfFuQf-LeKvSlud6x7jXYkG31#scrollTo=HOk9c4_35FKg)
 
 ## What types of research works?
-Anything! Remember, that this is just organized PyTorch code.
+Anything! Remember, that this is just an organized PyTorch code.
```
Comment (williamFalcon) [inline README.md:229]:
```suggestion
Anything! Remember, that this is just organized PyTorch code.
```

this is wrong...  (Use “a” before words that start with a consonant sound and “an” before words that start with a vowel sound). Py is not a vowel

Comment (Deeksha2501) [reply README.md:229]: but I had put "an" before organized(starting with a vowel)

Comment (justusschock) [reply README.md:229]:
```suggestion
Anything! Remember, that this is just organized PyTorch code.
```

IMO there is no article needed here...

File: README.md [hunk 3]
```diff
@@ -163,10 +163,10 @@ If your code IS a mess, then you needed to clean up anyhow ;)
 Although your research/production project might start simple, once you add things like GPU AND TPU training, 16-bit precision, etc, you end up spending more time engineering than researching. Lightning automates AND rigorously tests those parts for you.
 
 ## Support
-- [8 core contributors](https://pytorch-lightning.readthedocs.io/en/latest/governance.html) who are all a mix of professional engineers, Research Scientists, PhD students from top AI labs. 
+- [8 core contributors](https://pytorch-lightning.readthedocs.io/en/latest/governance.html) who are all a mix of professional engineers, Research Scientists, Ph.D. students from top AI labs. 
 - 100+ community contributors.
 
-Lightning is also part of the [PyTorch ecosystem](https://pytorch.org/ecosystem/) which requires projects to have solid testing, documentation and support.
+Lightning is also part of the [PyTorch ecosystem](https://pytorch.org/ecosystem/) which requires projects to have solid testing, documentation, and support.
```
Comment (justusschock) [inline README.md:169]:
```suggestion
Lightning is also part of the [PyTorch ecosystem](https://pytorch.org/ecosystem/) which requires projects to have solid testing, documentation and support.
```
Not needed before "and"

**Comment by williamFalcon** [comment]
Thank you @Deeksha2501  :)

**Comment by Deeksha2501** [comment]
looking forward to contributing more in open-source :)

**Pull request merged**